src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -638,12 +648,69 @@ describe('<OfferBody />', () => {
       })
     })
   })
+
+  describe('same artist playlist with "wipSameArtistPlaylist" feature flag activated', () => {
+    beforeAll(() => {
+      mockSameArtistPlaylist = mockedAlgoliaOffersWithSameArtistResponse
+    })
+
+    beforeEach(() => {
+      useFeatureFlagSpy.mockReturnValueOnce(false).mockReturnValueOnce(true)
+    })
+
+    it('should display same artist list when offer has some', async () => {
+      renderOfferBody({
+        sameArtistPlaylist: mockSameArtistPlaylist,
+      })
+
+      await screen.findByText('Envoyer sur Instagram')
+
+      expect(screen.queryByTestId('sameArtistPlaylist')).toBeOnTheScreen()
+    })
+
+    it('should not display same artist lists when offer has not it', async () => {
```

Comment:
`should not display same artist list when offer has not it`

Reply:
`has not it` ?
Si tu veux dire qu'on ne veut pas afficher de playlist pour une offre qui n'a pas d'artiste alors ce serait peut-être mieux `should not display same artist playlist if the offer has no artist`

Reply:
Ok pardon j'avais mal lu. Tu veux plutôt dire qu'on ne devrait pas afficher la playlist s'il n'y a pas d'offres concernées.
`should not display the playlist when there are no relevant offers. `